### PR TITLE
flamenco: use fd_vm_syscall_register_all

### PR DESCRIPTION
**Pull request by kbhargava-jump**

### CODE REVIEW

**Review by jumpsiegel — APPROVED** [review]

**Review by topointon-jump — COMMENTED** [review]
Awesome fuzzing find! Could you also add the fixture?

**Review by topointon-jump — DISMISSED** [review]

**Comment by ravyu-jump** [comment]
fuzzers <3

**Review by jumpsiegel — APPROVED** [review]

**Review by ravyu-jump — APPROVED** [review]

**Review by jumpsiegel — APPROVED** [review]

File: contrib/test/instr-fixtures/bpf-loader-upgradeable-v1-programs.list [hunk 1]
```diff
@@ -1 +1,199 @@
 dump/test-vectors/instr/fixtures/bpf-loader-upgradeable-v1-programs/instr-114U24XCn8Mxu5abKXM7XeYqETAb3wZdNAE9sRcZLqrHBM4tPDowAadcoPGg5zW75o3maq6s3N1WM4rdyw14jq-1207.fix
+dump/test-vectors/instr/fixtures/bpf-loader-upgradeable-v1-programs/058f9716ec2e93a78723d04cfbab526eeef2e448_1631513.fix
```
Comment (mjain-jump) [inline contrib/test/instr-fixtures/bpf-loader-upgradeable-v1-programs.list:2]: These should be moved to `bpf-loader-v3.list`

Comment (kbhargava-jump) [reply contrib/test/instr-fixtures/bpf-loader-upgradeable-v1-programs.list:2]: i used `-g` to group when creating fixtures and it put them under `bpf-loader-upgradeable-v1-programs`

**Pull request merged**